firebase_admin/db.py

```diff
@@ -678,8 +687,11 @@ def from_app(cls, app):
 
     def request(self, method, urlpath, **kwargs):
         resp_headers = kwargs.pop('resp_headers', False)
+        params = kwargs.get('params', None)
         resp = self._do_request(method, urlpath, **kwargs)
-        if resp_headers:
+        if resp_headers and params == 'print=silent':
```

Comment:
Ok. Lets leave it as it is for now. I'll try to think of a better way to refactor this bit out.